usecases/backup/coordinator.go

```diff
@@ -34,7 +35,7 @@ const (
 )
 
 var (
-	errNoShardFound = errors.New("no shard found")
+	// errNoShardFound = errors.New("no shard found")
```

Comment:
**suggestion (if-minor):** maybe we delete it given that we don't need it any more

Reply:
This is intentional to allow for easy revision once raft is merged to master.
The error will be used in the raft version.